bin/send_lhep8.py

```diff
@@ -143,15 +164,16 @@ def send(self, force):
             sys.exit(3)
 
         condor_file_str=''
+	
         for i in xrange(len(All_files)):
 
             if nbjobsSub == self.njobs: break
-
+            
             tmpf=None
             with open(All_files[i], 'r') as stream:
                 try:
                     #tmpf = yaml.load(stream, Loader=yaml.FullLoader)
-                    tmpf = yaml.load(stream)
+                    tmpf = yaml.load(stream, Loader=yaml.FullLoader)
```

Comment:
that depends on the python version, but should like this for newest ones, is this also changed to all the other places where yaml are loaded?